aiida/orm/implementation/sqlalchemy/nodes.py

```diff
@@ -441,13 +434,9 @@ def extras_keys(self):
         for key in self._dbmodel.extras.keys():
             yield key
 
-    def _flag_field(self, field):
-        from aiida.backends.sqlalchemy.utils import flag_modified
-        flag_modified(self._dbmodel, field)
-
-    def _flush_if_stored(self):
+    def _flush_if_stored(self, fields):
         if self._dbmodel.is_saved():
-            self._dbmodel.save()
```

Comment:
Will this new implementation work exactly the same for the SqlAlchemy backend? I.e. will `self._dbmodel._flush(fields)` do the same thing by calling `flag_modified` on all the fields?

Reply:
Ok, I think it actually does this in the `_flush` method of the `ModelWrapper` so looks to be all good

Reply:
Yes, this is also what I noticed. In both implementations the flag_modified and _dbmodel.save methods are called, so it should work exactly the same.